tests/conftest.py

```diff
@@ -2278,12 +2278,22 @@ def fixture_flow_cell_name() -> str:
     return "HVKJCDRXX"
 
 
+@pytest.fixture(name="expected_average_q30")
+def fixture_expected_average_q30() -> float:
+    """Return expected average Q30."""
+    return 90.5
+
+
 @pytest.fixture
 def store_with_sequencing_metrics(
```

Comment:
This is great that we have a store with the new table - is there a reason we don't add it to the default store fixture instead?